mpa/modules/datasets/composed_dataloader.py

```diff
@@ -46,6 +46,7 @@ def set_epoch(self, epoch):
     def __init__(self, loaders=[]):
         self.loaders = loaders
         self.max_iter = len(self.loaders[0])
+        self.dataset = [i.dataset for i in self.loaders]
```

Comment:
i.dataset is weird a little. could you rename i to loader..?

Reply:
Removed, thank you for suggestion.